tests/plugins/colorbutton/colorbutton.js

```diff
@@ -81,6 +101,48 @@
 				bgColorBtn.click( editor );
 				assert.areEqual( 0, bgColorBtn._.panel.getBlock( bgColorBtn._.id ).element.find( '.cke_colorauto' ).count(), 'Automatic button should not be visible.' );
 			} );
+		},
+
+		// 590
+		'test changing text color': function() {
+			var ed = this.editor,
+				bot = this.editorBot,
+				txtColorBtn = ed.ui.get( 'TextColor' ),
+				frame;
+
+			ed.on( 'dialogHide', function() {
+				assert.areEqual( '<h1><span style="color:#ff3333">Moo</span></h1>', ed.getData() );
+			} );
+
+			onDialogShow( ed );
+			bot.setHtmlWithSelection( '[<h1>Moo</h1>]' );
+
+			txtColorBtn.click( ed );
+			openColorDialog( frame );
+		},
+
+		// 590
+		'test changing text color with no focus': function() {
+			var editor = CKEDITOR.replace( 'editor' ),
```

Comment:
You could use `bender.editorBot.create` to create new editor (see other tests in this file).